cmd/gitops/ui/run/cmd.go

```diff
@@ -290,6 +306,31 @@ func runCmd(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func listenAndServe(srv *http.Server, options Options, log logrus.FieldLogger) error {
+	if options.NoTLS {
+		log.Info("TLS connections disabled")
+		return srv.ListenAndServe()
+	}
+
+	if options.TLSCert == "" && options.TLSKey == "" {
+		log.Info("TLS cert and key not specified, generating and using in-memory keys")
+
+		tlsConfig, err := wego_tls.TLSConfig([]string{"localhost", "0.0.0.0", "127.0.0.1"})
```

Comment:
This should probably fail if only one of them is provided?

Reply:
Yes!